### ISSUE TITLE
Slader Transition to Quizlet; extension no longer works

### ISSUE THREAD

**Issue by bribge** [lebr0nli/slader-extension#14]
At this point Slader will now fully redirect any link clicked to the Quizlet website, where the extension does not work. Changing VPN location does not seem to work. Incognito does, but is slow and inefficient. 

Has anyone found a work around? Is the extension being worked on to be compatible?

Thanks.

**Comment by EthanHarv** [comment]
Refer to #12

**Comment by bribge** [comment]
> Refer to #12

Yep just read through that, thanks. Is this project on hold until further notice?

**Comment by EthanHarv** [comment]
Yep. I'm still actively looking for more acceptable methods, so it's not impossible, but I cannot make any promises.

**Comment by bribge** [comment]
Gotcha. Seems that Quizlet has a much tighter control over what data they are releasing. Not to mention the UI is really a downgrade.

**Comment by Bluebuds0449** [comment]
https://youtu.be/m23Q_datSVM :- use this link and see the video to get for free
